### PR TITLE
Controllers specs

### PR DESCRIPTION
**Pull request by Goruchie**
# **Add the following features:**
- **Add RSpec as a gem to your project.**
- **Create a Request spec file for all your controllers.**
- **Make sure that for each action you check:**
- [x] If the response status was correct.
- [x] If a correct template was rendered.
- [x] If the response body includes correct placeholder text.

### CODE REVIEW

File: spec/requests/users_spec.rb [hunk 1]
```diff
@@ -1,7 +1,16 @@
...
+    before(:example) { get '/users' } # get(:index)
+    it 'is a success' do
+      expect(response).to have_http_status(:ok)
+    end
+    it "renders 'index' template" do
+      expect(response).to render_template('index')
+    end
+    it 'includes correct placeholder text' do
+      expect(response.body).to include('Show users:')
+    end
```
Comment (rotimiazeez) [inline spec/requests/users_spec.rb]: - Good job implementing the User `#index` action for this test 👍🏽. Kindly add a test for the User `#show` action to complete the test requirement 😉.

File: spec/requests/posts_spec.rb [hunk 2]
```diff
@@ -1,7 +1,17 @@
...
+
+    it 'is a success' do
+      expect(response).to have_http_status(:ok)
+    end
+    it "renders 'index' template" do
+      expect(response).to render_template('index')
+    end
+    it 'includes correct placeholder text' do
+      expect(response.body).to include('Show posts for a given user:')
+    end
```
Comment (rotimiazeez) [inline spec/requests/posts_spec.rb]: - Good job implementing the Post `#index` action for this test 👍🏽. Kindly add a test for the Post `#show` action to complete the test requirement 😉.

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @Goruchie 👍🏽,
## Good job so far!
### To Highlight 👇🏽
- No Linter error ✔️ 
- Good PR description 👍🏽 
- Good commit message(s) 😉
- Descriptive README 📖
You still need to work on some issues to prepare your project for the final evaluation, but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR WORDS>


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you consider them as they can improve your code._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag (@rotimiazeez) me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

**Comment by Goruchie** [comment]
@rotimiazeez Thanks for the review! I've implemented the required changes, and now my project should be approved.

**Review by Reem-lab — APPROVED** [review]
# Hi @Goruchie,👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- No Linters errors ✅
- Followed Gitflow ✅
- Great working on the tests✅
-  Professional README ✅
- Descriptive PR 💯💯


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Goruchie** [comment]
Thank you @Reem-lab!!

**Pull request merged**